modules/logging-loki-cli-install.adoc

```diff
@@ -18,28 +16,242 @@ To install and configure logging on your {product-title} cluster, additional Ope
-  namespace: openshift-operators-redhat <1>
+  namespace: openshift-operators-redhat # <1>
 spec:
-  channel: stable <2>
+  channel: stable # <2>
   name: loki-operator
-  source: redhat-operators <3>
+  source: redhat-operators # <3>
   sourceNamespace: openshift-marketplace
 ----
 <1> You must specify the `openshift-operators-redhat` namespace.
 <2> Specify `stable`, or `stable-5.<y>` as the channel.
 <3> Specify `redhat-operators`. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the `CatalogSource` object you created when you configured the Operator Lifecycle Manager (OLM).
 
-. Apply the `Subscription` object:
+. Apply the `Subscription` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create a `namespace` object for the {clo}:
++
+.Example `namespace` object
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+name: openshift-logging # <1>
+annotations:
+    openshift.io/node-selector: ""
+labels:
+    openshift.io/cluster-logging: "true"
+    openshift.io/cluster-monitoring: "true" # <2>
+----
+<1> The Red{nbsp}Hat OpenShift Logging Operator is only deployable to the `openshift-logging` namespace.
+<2> A string value that specifies the label as shown to ensure that cluster monitoring scrapes the `openshift-operators-redhat` namespace.
+
+. Apply the `namespace` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create an `OperatorGroup` object
++
+.Example `OperatorGroup` object
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging # <1>
+spec:
+  targetNamespaces:
+  - openshift-logging
+----
+<1> You must specify the `openshift-logging` namespace.
+
+. Apply the `OperatorGroup` object by running the following command:
 +
 [source,terminal]
 ----
 $ oc apply -f <filename>.yaml
 ----
+
+. Create a `Subscription` object:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging # <1>
+spec:
+  channel: stable # <2>
+  name: cluster-logging
+  source: redhat-operators # <3>
+  sourceNamespace: openshift-marketplace
+----
+<1> You must specify the `openshift-logging` namespace.
+<2> Specify `stable`, or `stable-5.<y>` as the channel.
+<3> Specify `redhat-operators`. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the CatalogSource object you created when you configured the Operator Lifecycle Manager (OLM).
+
+. Apply the `Subscription` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+
+. Create a `LokiStack` CR:
++
+.Example `LokiStack` CR
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
```

Comment:
It was recommended to set in the examples how to set the Loki Retention. At least global as it not retention is set, the chunks are never deleted impacting in getting object storage full and this has a big impact. Then, it shouldn't exist an example where at least gobal retention and per stream is present as visible in https://docs.openshift.com/container-platform/4.17/observability/logging/log_storage/cluster-logging-loki.html#logging-loki-retention_cluster-logging-loki